src/clients/aggregator_client/api_types.rs

```diff
@@ -101,12 +101,34 @@ impl From<Option<i64>> for QueryType {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct AuthenticationToken {
+    /// Type of the authentication token. Always "Bearer" in divviup-api.
+    #[serde(rename = "type")]
+    typ: String,
```

Comment:
Nit:
```suggestion
    r#type: String,
```
avoids having to rename (and the awkward `typ`)

If you don't like that, maybe call it `token_type` in rust and rename to "type" in serde?

Reply:
Oh, cool, I didn't know about this bit of syntax. I'm happy to use `r#type` in this project if you like it better.